### PR TITLE
add Netty arm64 dependencies

### PR DESCRIPTION
**Pull request by sullis**
## Description

add Netty arm64 dependencies (aarch_64)

/nocl

## Motivation and Context

improve compatibility with ARM 64 systems

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-server/src/test/java/org/graylog2/NettyTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package org.graylog2;
```
Comment (bernd) [inline graylog2-server/src/test/java/org/graylog2/NettyTest.java:1]: Please run `./mvnw license:format` to add the missing license headers. Thanks!

**Review by bernd — CHANGES_REQUESTED** [review]
Thank you for your contribution! One small comment below.

File: graylog2-server/src/test/java/org/graylog2/NettyTest.java (hunk 1 above)
Comment (sullis) [reply graylog2-server/src/test/java/org/graylog2/NettyTest.java:17]: Done.

**Comment by sullis** [comment]
`workflows awaiting approval`

**Review by bernd — APPROVED** [review]
Thank you! :pray:

**Pull request merged**